README.md

```diff
@@ -33,11 +36,6 @@ This plugin helps you to read [Omnivore](https://omnivore.app/) articles and hig
 
 ![fetching](./fetching.gif)
 
-## Roadmap
-
-- [ ] Refactor the plugin to use the new Omnivore API
-- [ ] Saving PDFs of the Omnivore article
-
 ## Contacts
```

Comment:
We could also remove this part. It is from my old repo 😆

Reply:
hah, i want you to get all bug reports!